Source/FFmpegMediaSource.cpp

```diff
@@ -353,7 +353,7 @@ namespace winrt::FFmpegInteropX::implementation
 
         for (auto& stream : subtitleStreams)
         {
-            stream->PlaybackItemWeak = playbackItem;
+            stream->PlaybackItem = playbackItem;
```

Comment:
This change is not right. It is undoing part of the auto ref PR. We need weak ref here to avoid circle reference.